kubernetes-image/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+ORG_PATH=github.com/c-fs
+REPO_PATH=$ORG_PATH/cfs
+
+go build -a -tags netgo -installsuffix netgo --ldflags '-extldflags "-static"' -o cfs ${REPO_PATH}/server
+docker build -t yunxing/cfs:k8s .
```

Comment:
why use `yunxing/cfs:k8s` here? Shouldn't we use master? If we need something from Yunxing's branch, could we merge that change into master?